GoogleUtilities/Environment/SecureStorage/GULKeychainStorage.m

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-#import "GoogleUtilities/Environment/Public/GULKeychainStorage.h"
+#import "GoogleUtilities/Environment/Private/GULKeychainStorage.h"
```

Comment:
I wonder if there is a way to implement test apps like #5828 with this structure.

Reply:
I don't think that test apps should depend on the repo structure. My intent is to only change the library sources and unit test imports.

Reply:
The issue (existing one already) that there are no actual public header in GoogleUtilities/Environment (they are marked as private now). I was going to make them public. Do you think it may be an issue?

Reply:
Yes. We should avoid public headers since they require ifdef'ing between Swift Package Manager and CocoaPods.  See the Logger and NSData changes in this PR for details.

Reply:
It feels a bit weird to have a library without public API, but it's probably out of scope of the PR, we may discuss it later